### ISSUE TITLE
Bug: Can't switch input currencies multiple times 

### ISSUE THREAD

**Issue by TorstenStueber** [pendulum-chain/vortex#170]
The input currency can only be changes once, after that the button for selecting the currency does not have any effect anymore.

## To reproduce
* click on the currency selector on the input field and change the currency
* try to click the selector again, the popup with the selector will not show up

The following error show up after changing the currency for the first time, it is possibly related:

```
Uncaught (in promise) Error: [big.js] Invalid number
    at parse$5 (index-MMiAsjmI.js:1407:103986)
    at new ee (index-MMiAsjmI.js:1407:103745)
    at ee (index-MMiAsjmI.js:1407:103539)
    at k$4.SwapPage [as constructor] (index-MMiAsjmI.js:1489:90869)
    at k$4.q$5 [as render] (index-MMiAsjmI.js:1:10955)
    at L$2 (index-MMiAsjmI.js:1:8289)
    at w$4 (index-MMiAsjmI.js:1:2804)
```

**Comment by prayagd** [comment]
Should we close this as this fixed int he PR #174 ?

**Comment by gianfra-t** [comment]
Yes! Solved.
